modules/dreamview/frontend/src/store/websocket/websocket_realtime.js

```diff
@@ -16,7 +16,8 @@ export default class RealtimeWebSocketEndpoint {
         this.routingTime = undefined;
         this.currentMode = null;
         this.worker = new Worker();
-
+        this.cameraWS = null;
```

Comment:
Seems like the change for this.cameraWS is half way. Can we either finish it or remove the reference here?

Reply:
will remove it for now. will fix later.